lib/core/src/lib/notifications/components/notification-history.component.html

```diff
@@ -6,10 +6,8 @@
             class="adf-notification-history-menu_button"
             id="adf-notification-history-open-button"
             (menuOpened)="onMenuOpened()">
-        <mat-icon matBadge="&#8288;"
-                  [matBadgeHidden]="!notifications.length"
-                  matBadgeColor="accent"
-                  matBadgeSize="small">notifications</mat-icon>
+            <mat-icon *ngIf="notifications.length" svgIcon="notification:pending_notifications"></mat-icon>
```

Comment:
should be `<adf-icon>` if you register custom images

Reply:
made changes to the existing code, didnt have to use adf-icon